packages/typebox/test/index.test.ts

```diff
@@ -81,7 +81,7 @@ describe('@feathersjs/schema/typebox', () => {
   })
 
   it('defaultAppConfiguration', async () => {
-    const configSchema = Type.Composite([
+    const configSchema = Type.Intersect([
```

Comment:
I think the preferred version in v1 is `Interface` and `Evaluate`

https://github.com/sinclairzx81/typebox/blob/main/changelog/1.0.0-migration.md#typecomposite

Reply:
I think I got it working with help from @sinclairzx81 in https://github.com/sinclairzx81/typebox/issues/1361